### PR TITLE
Not https

### PR DESCRIPTION
**Pull request by PaulWessel**
The link I tried was http only but I added https to the index file - hence failure.

### CODE REVIEW

**Comment by meghanrjones** [comment]
Is https not supported for this mirror?

**Comment by PaulWessel** [comment]
It does, but that is the real URL at https://opentopography.s3.sdsc.edu/minio/gmtdata/ 
I think all the generic-mapping-tools forwards except oceania are http but many of them point to https real URLs.

**Comment by PaulWessel** [comment]
Not that I fully understand this, mind you, but it seems we are just redirecting from our domain to either http or https depending on what the mirror has been set up to support.  So in this case, https.

File: mirrors/index.rst [hunk 1]
```diff
@@ -65,7 +65,7 @@ to the mirror that is closest to you to minimize transmission times.
      - U of Sci. & Tech. of China, China
      - .. image:: https://img.shields.io/website?down_message=offline&label=%20&style=plastic&up_message=OK&url=http%3A%2F%2Fchina.generic-mapping-tools.org/gmt_data_server.txt
    * - **OpenTopography/SDSC** [US West Coast]
-     - https://sdsc-opentopography.generic-mapping-tools.org
+     - http://sdsc-opentopography.generic-mapping-tools.org
      - OpenTopography at San Diego Supercomputing Center
      - .. image:: https://img.shields.io/website?down_message=offline&label=%20&style=plastic&up_message=OK&url=https%3A%2F%2Fsdsc-opentopography.generic-mapping-tools.org/gmt_data_server.txt
```
Comment (meghanrjones) [inline mirrors/index.rst:70]:
```suggestion
     - .. image:: https://img.shields.io/website?down_message=offline&label=%20&style=plastic&up_message=OK&url=http%3A%2F%2Fsdsc-opentopography.generic-mapping-tools.org/gmt_data_server.txt
```

Comment (PaulWessel) [reply mirrors/index.rst]: OK done I think

Comment (meghanrjones) [reply mirrors/index.rst]: It's still showing up as offline despite the redirect working: https://website-iksg35ayw-gmt.vercel.app/mirrors/

**Comment by PaulWessel** [comment]
Maybe there is a time lag on those checks?

**Review by meghanrjones — APPROVED** [review]

**Pull request merged**